### PR TITLE
HEEDLS-NONE Remove some redundant test helper methods

### PR DESCRIPTION
**Pull request by benjimarshall**
## Changes
As promised in the previous PR: rearrange the helper functions in the various data test helpers so each helper relates to a specific concept (tutorial, course, etc.) and remove a redundant helper which had two implementations.

## Testing
All tests pass

### CODE REVIEW

File: DigitalLearningSolutions.Data.Tests/Helpers/TutorialContentTestHelper.cs [hunk 1]
```diff
@@ -65,5 +95,14 @@ public void UpdateTutorialStatus(int tutorialId, int customisationId, int status
                 new { tutorialId, customisationId, status }
             );
         }
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Data.Tests/Helpers/TutorialContentTestHelper.cs:97]: Empty line between functions please. And might be worth moving this up to the other diagnostic functions? 🙂

**Review by FridaTveit — COMMENTED** [review]
One small thing, otherwise looks good 🙂

File: DigitalLearningSolutions.Data.Tests/Helpers/TutorialContentTestHelper.cs (hunk 1 above)
Comment (benjimarshall) [reply DigitalLearningSolutions.Data.Tests/Helpers/TutorialContentTestHelper.cs:97]: Yep sorry! Rearranged in 60d7021 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**